packages/arb-token-bridge-ui/src/pages/api/isAddressDenylisted.ts

```diff
@@ -0,0 +1,38 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import denylist from '../../../public/__auto-generated-denylist.json'
+
+type Request = NextApiRequest & { query: { address: string } }
+
+export default async function handler(
```

Comment:
We could change this api route to better follow a REST architecture:

```
/api/denylist?address={address}
```

We already follow this in our other routes:

<img width="146" alt="Screenshot 2023-07-06 at 10 39 00" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/20543771/fc79a93c-3b29-423b-983c-b9e9dfc14b00">

Great post on the topic: https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#restful